core/parameters/src/config_store.rs

```diff
@@ -136,6 +138,18 @@ impl RuntimeConfigStore {
                 let genesis_runtime_config = RuntimeConfig::initial_testnet_config();
                 Self::new(Some(&genesis_runtime_config))
             }
+            near_primitives_core::chains::BENCHMARKNET => {
+                let mut config_store = Self::new(None);
+                let mut config = RuntimeConfig::clone(config_store.get_config(PROTOCOL_VERSION));
+                config.congestion_control_config.max_tx_gas = 10u64.pow(8);
```

Comment:
I think a good value for max and min gas would be 100 PGas, so higher than the current value.

Reply:
Actually, we can also use an old "unlimited" value: 9_223_372_036_854_775_807

Reply:
Btw I seen high tps on gcp without changing this values